### PR TITLE
Add Ballerina Greeter Service Sample

### PR DESCRIPTION
**Pull request by Asitha**
relates to https://github.com/wso2-enterprise/choreo/issues/20447

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: ballerina/greeter/.devcontainer/.devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Ballerina Dev Container",
```
Comment (Mirage20) [inline ballerina/greeter/.devcontainer/.devcontainer.json:2]:
```suggestion
    "name": "Ballerina Greeter",
```

File: ballerina/greeter/tests/service_test.bal [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import ballerina/io;
```
Comment (Mirage20) [inline ballerina/greeter/tests/service_test.bal:1]: Do we need this test file? 
I think we can remove it since this is a small sample.

Comment (Asitha) [reply ballerina/greeter/tests/service_test.bal:1]: Let's keep it :) It's the boilerplate tests that come with the Ballerina greeter template

Comment (Mirage20) [reply ballerina/greeter/tests/service_test.bal:1]: If we run the test, it will fail, right? because the server is not running?

**Comment by Mirage20** [comment]
Shall we add some instructions to the readme file? 
Example: https://github.com/wso2/choreo-sample-apps/blob/main/go/rest-api/README.md

File: ballerina/greeter/tests/service_test.bal (hunk 2 above)
Comment (Asitha) [reply ballerina/greeter/tests/service_test.bal:1]: `bal test` works!

**Review by Mirage20 — APPROVED** [review]

**Pull request merged**